WhaleSpotting/Controllers/WhaleSightingController.cs

```diff
@@ -27,4 +29,23 @@ public IActionResult GetById([FromRoute] int Id)
             return NotFound();
         }
     }
+    [HttpGet("search")]
+    public IActionResult Search([FromQuery] WhaleSightingSearchRequest whaleSightingSearchRequest)
+    {
+        try
+        {
+            var whaleSightings = _whaleSightingService.Search(whaleSightingSearchRequest);
+            List<WhaleSightingResponse> whaleSightingResponses= new List<WhaleSightingResponse>();
+            foreach (WhaleSighting whaleSighting in whaleSightings)
+            {
+                whaleSightingResponses.Add(new WhaleSightingResponse(whaleSighting));
+            }
```

Comment:
We can return a `<List<WhaleSightingResponse>>` directly from the `_whaleSightingService.Search(whaleSightingSearchRequest)` method so we can avoid this, I will add more further down